src/Metadata/ValueObject/Column.php

```diff
@@ -68,19 +68,31 @@ public function __construct(
         }
 
         if ($sanitizedName === '') {
-            throw new InvalidArgumentException('Column\'s sanitized name cannot be empty.');
+            throw new InvalidArgumentException(sprintf(
+                'Column\'s %s sanitized name cannot be empty.',
```

Comment:
Upravil som aj exception msg, aby ak by bol este nejaky problem, tak aby sme vedeli ktoreho stlpca sa to tyka.

Reply:
dej ho pls do uvozovek - `Column\'s "%s" sanitized name cannot be empty.`